docs/content/about.md

```diff
@@ -1,22 +1,26 @@
 # Overview
 
-CDC Connectors for Apache Flink<sup>®</sup> is a set of source connectors for <a href="https://flink.apache.org/">Apache Flink<sup>®</sup></a>, ingesting changes from different databases using change data capture (CDC).
-The CDC Connectors for Apache Flink<sup>®</sup> integrate Debezium as the engine to capture data changes. So it can fully leverage the ability of Debezium. See more about what is [Debezium](https://github.com/debezium/debezium).
+CDC Connectors for Apache Flink<sup>®</sup> is a set of source connectors for <a href="https://flink.apache.org/">Apache
```

Comment:
It looks like this file is modified massively (maybe auto formatting?). What about only adding Vitess related contents, and create another PR if you do wanna reformat the file?